device/iot-device-client/src/main/java/com/microsoft/azure/sdk/iot/device/ClientType.java

```diff
@@ -1,12 +1,11 @@
-package com.microsoft.azure.sdk.iot.device.transport.amqps;
```

Comment:
Not sure why this ever lived in the amqp package considering it was only used in the device client level package "com.microsoft.azure.sdk.iot.device"